timesketch/lib/analyzers/sigma_tagger.py

```diff
@@ -69,14 +77,22 @@ def run(self):
                     rule.get('es_query'), rule.get('file_name'),
                     tag_list=rule.get('tags'))
                 tags_applied[rule.get('file_name')] += tagged_events_counter
+                time.sleep(0.5)
```

Comment:
why this sleep? is that really necessary, if a sleep is necessary then please document why, and why the 0.5?